src/tools/ogmaTool.ts

```diff
@@ -48,8 +48,10 @@ export class OgmaTools {
       return true;
     }
     const hexRegExp = /#[A-Fa-f0-9]{3,6}/;
-    const rgbRegExp = /^rgb\(\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*\)$/i;
-    const rgbaRegExp = /^rgba\(\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*(?:0|1|0?\.\d+)\s*\)$/i;
+    const rgbRegExp =
+      /^rgb\(\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*,\s*([01]?\d\d?|2[0-4]\d|25[0-5])\s*\)$/i;
+    const rgbaRegExp =
```

Comment:
@aminerhanemi I think the main change in the PR is the regex right ?

Reply:
no, this line is just some formatting. In the end I didn't change the regex